### PR TITLE
doc: Add example about time-range option

### PR DESCRIPTION
**Pull request by yujeongKim**
Hi! @namhyung 

I add example for uftrace-replay.md. This mentioned at [fd137a56b0](https://github.com/namhyung/uftrace/commit/fd137a56b0)

What do you think about it ? Thank you!

### CODE REVIEW

File: doc/uftrace-replay.md [hunk 1]
```diff
@@ -242,6 +242,34 @@ Each field has following meaning:
 
 The default value is 'duration,tid'.  If given field name starts with "+", then it'll be appended to the default fields.  So "-f +time" is as same as "-f duration,tid,time".  And it also accepts a special field name of 'none' which disables the field display and shows function output only.
 
+EXAMPLE
+========
+This command's default fields are DURATION and TID.
+
+When using time-range ```-r``` option by default, you can see TIMESTAMP or ELAPSED fields also.
```
Comment (namhyung) [inline doc/uftrace-replay.md]: You can use single backquote for this short description.

**Comment by yujeongKim** [comment]
Thank you for reviewing! :) I modifed!

**Comment by namhyung** [comment]
You need to combine two patches into a single patch and resubmit, thanks.

File: doc/uftrace-replay.md [hunk 2]
```diff
@@ -177,6 +177,32 @@ You can also see replay output with different time threshold for the same record
        6.448 us [ 1234] |   a();
        8.631 us [ 1234] | } /* main */
 
+In addition, The `-r` option can show functions executed within the given time range.
+When using this option by default, you can see TIMESTAMP or ELAPSED fields as well as DURATION and TID.
+```
```
Comment (namhyung) [inline doc/uftrace-replay.md]: Please use indentation (of 4 spaces) instead of the triple backtick for code block for consistency.

**Pull request closed without merging**